MH-FiorellaTheme5/source/stylesheets/styles.css

```diff
@@ -717,4 +813,28 @@ p{
   .social-media{
     border: 0px;
   }
+
+  .title-icon{
+    top: 50px;
+  }
+
+  .promotion-message{
+    left: 60px;
+    bottom: 35px;
+    font-size: 50px;
+  }
+
+  .box-promotions{
+    height: 500px;
+    background-image: url('../images/home/banner_image_home_1024@2x.jpg');
+  }
+
+  .p-prom-description{
```

Comment:
1. missing space before `{`
2. Please reorder the properties in alphabetical order